backend/presentation/services.py

```diff
@@ -0,0 +1,60 @@
+from typing import OrderedDict, List
+
+from presentation.models import Presentation, Tag
+
+
+PresentationData = List[OrderedDict[str, str]]
```

Comment:
If using python 3.9, you don't have to import built-in collections.
```suggestion
PresentationData = list[OrderedDict[str, str]]
```

Reply:
I've meant a data structure that would spare you verobse `.get("property", "")` later.